src/components/expenses/expensesSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {Text, View, StyleSheet} from 'react-native';
+import {ExpensePeriod} from '../../services/expenses/interfaceExpense';
+
+const ExpensesSummary = ({expensesPeriod, expenses}: ExpensePeriod) => {
+  const expensesSum = expenses.reduce((sum, expense) => {
+    return sum + expense.amount;
+  }, 0);
+  return (
+    <View style={styles.container}>
+      <Text style={styles.period}>{expensesPeriod}</Text>
+      <Text style={styles.sum}>R$ {expensesSum.toFixed(2)}</Text>
+    </View>
+  );
+};
+
+export default ExpensesSummary;
+
+const styles = StyleSheet.create({
+  container: {
+    padding: 8,
+    paddingHorizontal: 16,
+    backgroundColor: '#e4d9fd',
```

Comment:
Try export this color to a Color System file. Will be more flexible to not repeat  your code.